modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift4Codegen.java

```diff
@@ -59,7 +59,7 @@
     protected static final String LIBRARY_RX_SWIFT = "RxSwift";
     protected static final String[] RESPONSE_LIBRARIES = {LIBRARY_PROMISE_KIT, LIBRARY_RX_SWIFT};
     protected String projectName = "OpenAPIClient";
-    protected boolean unwrapRequired;
+    protected boolean unwrapRequired = true;
```

Comment:
@joeboyscout04 thanks for the fix. Since the default was "false", we'll need to target this in 4.0.x (upcoming major release)

If you change it back to "false" as the default, we can merge it into master (which only allows backward-compatible changes)

Reply:
Unless I'm misunderstanding what this is supposed to do, it is already unwrapping the required parameters by default.  I think this option never actually did anything.

Reply:
I mean unwrapRequired defaults to false at the moment. Changing the default setting of `unwrapRequired` to "true" will impact users who do not need this feature. 

(I didn't say your PR does not fix the issue and I agree with you the option may have been broken as it's not switched on by default)